API/ev3_sensors/nxt_temperature.c

```diff
@@ -0,0 +1,40 @@
+#include "ev3_inputs/ev3_input_iic.h"
+#include "ev3_sensors/ev3_sensors.h"
+#include "nxt_temperature.h"
+
+
+SensorHandler * NXTTemperature = &(SensorHandler) {
+    .Init = initNXTTemperatureSensor,
+    .Exit = exitNXTTemperatureSensor,
+    .currentSensorMode = {NONE_MODE, NONE_MODE, NONE_MODE, NONE_MODE}
+};
+
+bool initNXTTemperatureSensor (int port) {
+    NXTSound->currentSensorMode[port] = NXT_TEMPERATURE_SENSOR_DEFAULT_MODE;
+    return true;
+}
+
+// TODO: Not tested yet
+int ReadNXTTemperatureSensor(int port, NXTTemperatureUnit unit) {
```

Comment:
I have found that in the past, I have deciphered the bit operations into a more readable form:
https://github.com/JakubVanek/EV3-API/blob/23313766ee7689cc54e12496df85036f4e9a135a/API/ev3sensor_dispatch.c#L81

It seems similar, however in that code I'm missing `(temp>>4) & 0xFF` that is present here.